terraform/global-resources/locals.tf

```diff
@@ -0,0 +1,9 @@
+locals {
```

Comment:
MoJ have tagging guidelines that we should follow - https://ministryofjustice.github.io/technical-guidance/documentation/standards/documenting-infrastructure-owners.html#tags-you-should-use